### ISSUE TITLE
should chocolate-strife include the original intro video ?

### ISSUE THREAD

**Issue by Rathmox** [chocolate-doom/chocolate-doom#1626]
Strife originally shipped with a video in AVI format loaded with a DOS AVI binary with a batch script.

To keep the original experience, should chocolate-strife be able to play the video, even if originally the video wasn't played by the engine ?

The video I'm talking about: https://youtu.be/D2swXzKPSVA

**Comment by fabiangreffrath** [comment]
I don't think it adds much to the game, but it would be an ingesting exercise to implement support for it. Do you know in which file the video is contained?

**Comment by Rathmox** [comment]
The file shipped with Strife Veteran Edition (at least the GOG version).

it's in `[Game folder]/game/DOS/C/STRIFE/strife.avi`

The batch script is `strife.bat`

The format is quite wierd, VLC doesn't really read it, but mpv does. Converting it may be useful

**Comment by Sonicthefan** [comment]
yes please
